File: yyxarch/src/main/java/com/yyxnb/yyxarch/annotation/LceeStatus.java
package com.yyxnb.yyxarch.annotation;

import android.support.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@IntDef({LceeStatus.Loading, LceeStatus.Content, LceeStatus.Empty, LceeStatus.Error})
@Retention(RetentionPolicy.SOURCE)
public @interface LceeStatus {
    int Loading = 1;
    int Content = 2;
    int Empty = 3;
    int Error = 4;
}

File: yyxarch/src/main/java/com/yyxnb/yyxarch/annotation/LceeStatus.kt
package com.yyxnb.yyxarch.annotation

import androidx.annotation.IntDef
import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

@IntDef(LceeStatus.Loading, LceeStatus.Content, LceeStatus.Empty, LceeStatus.Error)
@Retention(RetentionPolicy.SOURCE)
annotation class LceeStatus {
    companion object {
        const val Loading = 1
        const val Content = 2
        const val Empty = 3
        const val Error = 4
    }
}
